src/TestFabric.java

```diff
@@ -37,8 +36,7 @@ public static void main(String[] args) throws InterruptedException {
 		
 		// Scroll down to where you can see the "View all sessions" button
 		WebElement viewAllSessions = driver.findElement(By.linkText(CRASHLYTICS_VIEW_ALL_SESSIONS_LINK));
-		Locatable element = (Locatable) viewAllSessions;
-		Point p = element.getCoordinates().onPage();
+		Point p = viewAllSessions.getLocation();
```

Comment:
Is there a particular version of Selenium web driver / Chrome where you are seeing this exception ?

Reply:
I used the latest version available.

Reply:
Version 3.8.1 from [here](http://www.seleniumhq.org/download/).

Reply:
@krshmbb Do you still have open questions?

Reply:
Nope. Merged. Thanks for pushing up the fix!